shimmingtoolbox/optimizer/basic_optimizer.py

```diff
@@ -71,12 +71,15 @@ def optimize(self, unshimmed, mask, mask_origin=(0, 0, 0), bounds=None):
 
         mx, my, mz = mask_origin
         mX, mY, mZ = mask.shape
+        mV = mX * mY * mZ
```

Comment:
might be out of scope for this PR but this should probably also be snake-case, right?

Reply:
Did I forget to change those? I totally did. Changing that and then squashing